addon/mixins/route.js

```diff
@@ -245,8 +262,8 @@ export default Ember.Mixin.create({
    @return {Object}
    */
   _includeBoundParams: function(options, boundParams) {
-    if (Ember.keys(boundParams).length > 0) {
-      Ember.keys(boundParams).forEach( (key) => {
+    if (keys(boundParams).length > 0) {
+      keys(boundParams).forEach( (key) => {
```

Comment:
Just a style issue, but parens around `(key)` are not strictly needed. You could even one-line it if you wanted to

```js
keys(boundParams).forEach(k => options[k] = this.get(boundParams[k])
```